Player.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+
+
+namespace Card_Game
+{
+    public class Player 
+    {
+        bool isPlayerInHand = false;  // na true, jeśli prowadzi grę
+        bool isPlayerInGame = false; // na true, jeśli jeszcze w grze
```

Comment:
it could be a property public get; private set; (then you won't need methods to get this value)

)